sources/packages/web/src/components/common/OfferingFormSubmit.vue

```diff
@@ -156,6 +156,8 @@ export default defineComponent({
             warningsTypes,
           );
         }
+        // Update the validation date from the API result.
+        lastCalculationDate = validationResult.validationDate;
```

Comment:
Thanks for adding the missing assignment back and fixing the code warning/error.